MetaMask/metamask-mobile
# feat(2808): improvements-and-small-features-and-small-fixes-that-still-needed-to-be-added-to-edit-permissions

## **Description**
Adding missing features, small fixes and feedback from design QA.


## **Related issues**

Fixes:

## **Manual testing steps**

1. set the feature flag export MM_MULTICHAIN_V1_ENABLED="1"
2. go to the dapp browser and open pancake swap and click Connect in pancake swap 
3. change the permissions and check fixes and improvements in the commit names are present

## **Screenshots/Recordings**



| Before      | After      |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/bdfdc4d5-3ecc-4da6-95f2-25f762cefe0f"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/503a1925-d837-4eb2-a02d-3990c7649e81"> |


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
